packages/zoe/src/contracts/coveredCall.js

```diff
@@ -16,42 +16,48 @@ import harden from '@agoric/harden';
 // extent of the invite:
```

Comment:
`the owner of the digital asset's offer exitRules must be of the kind "afterDeadline".` should be updated.